tern/utils/container.py

```diff
@@ -92,7 +92,7 @@ def build_container(dockerfile, image_tag_string):
     if not check_image(image_tag_string):
         try:
             client.images.build(path=path, tag=image_tag_string, nocache=True)
-        except(TypeError, docker.errors.APIError, docker.errors.BuildError):
+        except (TypeError, docker.errors.APIError, docker.errors.BuildError):  # pylint: disable=try-except-raise
```

Comment:
Could you suggest another way of raising an error that includes the error message? Maybe:
```
try:
    client.images.build...
except (TypeError, docker.errors.APIError, ...) as e:
    raise type(e)
```
?

Reply:
I'm not sure what your intent is here?  Do you want to raise a general TernError exception but pull the message from the exception raised?

Reply:
No, I want to bubble-up the caught exception whether it is `TypeError` or one of the `docker.errors` with the associated error message.